vcpkg/get_started/get-started-vscode.md

```diff
@@ -158,20 +158,20 @@ you'll need to provide the `vcpkg.cmake` toolchain file. To automate this,
 create a `CMakePresets.json` file in the "helloworld" directory with the
 following content:
 
+
 ```json
 {
-    "version": 3,
-    "configurePresets": [
-        {
-            "name": "vcpkg",
-            "cacheVariables": {
-                "CMAKE_TOOLCHAIN_FILE": {
-                   "value": "$env{VCPKG_ROOT}/scripts/buildsystems/vcpkg.cmake",
-                    "type": "FILEPATH"
-                }
-            }
-        }
-    ]
+  "version": 1,
```

Comment:
The VS Code CMake Tools extension only supports version 2 or later for `CMakePresets.json`.